starter_code/views/layout.hbs

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="/stylesheets/styles.css">
+</head>
+
+<body>
+  <nav>
+
+    <div><a class="nav-link" href="/">Home</a>
```

Comment:
You could remove theses divs because they don't do anything for your code. Then you can remove the class and target it with just the tags 

```
nav a {
  text-decoration: none;
  color: white;
  margin-left: 40px;
} 
```